src/api/items.js

```diff
@@ -14,3 +15,9 @@ export const getItemsDetails = (id, token) => async (dispatch) => {
     .then((response) => response.json())
     .then((res) => dispatch(fetchItemsDetails(res.data)));
 };
+
+export const deleteItemFromAPI = (id) => async (dispatch) => {
+  await axios.delete(`${BASE_URL}/${id}`)
+    .then(() => ({ status: `Item#${id} Deleted successfully` }))
+    .then((res) => dispatch(deleteItemFromState(res)));
+};
```

Comment:
**[REQUIRED]** Please can you add a token while making an API call to delete an item. This will serve as an authorization to enable you to access that endpoint.

Reply:
Thanks for your review,

Sadly i tried to test the front end authentication feature and it didn't work on my end.

Maybe you can help me and add that line and test if its working. 

Thanks.